sklearnex/linear_model/incremental_linear.py

```diff
@@ -103,6 +103,13 @@ class IncrementalLinearRegression(
     n_features_in_ : int
         Number of features seen during ``fit`` or ``partial_fit``.
 
+    Note
```

Comment:
Since this note will appear in multiple classes, perhaps it could be moved into a common variable and the docstrings modified programmatically for all the classes that will ahve the note.

Reply:
It is good idea. I think it could be done in a separate PR moving some common things for all incremental algorithms to separate classes/variables.